### DISCUSSION TITLE
Thank you for building this

### DISCUSSION THREAD

**Discussion by insanebaba** [cornucopia-rs/cornucopia#145] [General]
I was looking for such a tool, similar to sqlc in golang. 

So thank you for making this.
It's way better than ORMs.

Thank you again ☺️

**Comment by LouisGariepy** [comment]
Thanks! This means a lot to us.

Personally, it makes my day to know people are finding the crate useful, and my hope is to keep making it better over time, thanks to our awesome users and contributors!

So, thanks again for your kind words. I'll move this to a discussion so others can chime in if they want to :smile: .

**Comment by jacobsvante** [comment]
I have to chime in and say YES, thank you so much for putting all the effort into building this! It has such a nice simplicity to it.

I've been looking for a "query builder" that gets it right in Rust. I've tried out both sqlx and sqlx + sea-query, but they just don't feel quite right to me.

**Comment by ianpurton** [comment]
Thanks from me too. This has transformed the way I work with the database.

Every language should have a cornucopia.
